tests/test.rs

```diff
@@ -243,3 +243,48 @@ fn default_padding() {
 
     assert_eq!(v.0, 0x8fff_00ff);
 }
+
+#[test]
+fn default_endianness() {
```

Comment:
Yes, tests are important, but the default one might be a bit redundant.

Reply:
my thought was to have an explicit test for default behavior, but I guess that's covered like twenty times over by the other tests eh :sweat_smile: